src/gscreenshot/screenshooter/scrot.py

```diff
@@ -52,7 +52,7 @@ def can_run() -> bool:
         """Whether scrot is available"""
         try:
             scrot_version_output = subprocess.check_output(['scrot', '--version'])
-            scrot_version_str = scrot_version_output.decode().strip().rsplit(' ', maxsplit=1)[-1]
+            scrot_version_str = scrot_version_output.decode().strip().rsplit('.', maxsplit=1)[-1]
             scrot_major_version = scrot_version_str.split('.')[0]
```

Comment:
Hey there, thanks! With this change, this line (56) isn't really needed and we can just compare off of the scrot_version_str variable.

```
>>> foo = "scrot version 1.8.1".strip().rsplit('.', maxsplit=1)
>>> print(foo)
['scrot version 1.8', '1']
>>> foo = "scrot version 1.8.1".strip().rsplit('.', maxsplit=1)[-1]
>>> print(foo)
1
>>> print(foo.split('.')[0])
1
```

Reply:
Fixed!